pyexamples/wikistream.py

```diff
@@ -23,22 +24,19 @@ def gen_input():
         yield event.data
 
 
-def server_name(data_dict):
-    return data_dict["server_name"]
-
-
-def count_edits(acc, server_names):
-    for server_name in server_names:
-        acc[server_name] += 1
-    return acc
+def initial_count(data_dict):
+    return data_dict["server_name"], 1
 
 
 ec = bytewax.Executor()
-flow = ec.Dataflow(inp.tumbling_epoch(2.0, gen_input()))
+flow = ec.Dataflow(inp.tumbling_epoch(2.0, open_stream()))
+# "event_json"
 flow.map(json.loads)
-flow.map(server_name)
-flow.exchange(hash)
-flow.accumulate(lambda: collections.defaultdict(int), count_edits)
+# {"server_name": "server.name", ...}
+flow.map(group_by_server)
```

Comment:
Is this `group_by_server` missing?

Reply:
Oops. That's supposed to be the `initial_count` from above. Fixed.